index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Retky Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <header>
+      <div class="statusBar">
+        <div id="clock">
+          <p>9:41</p>
+        </div>
+        <div class="status">
+          <img src="img/CellularConnection.png" alt="Cellular Connection">
+          <img src="img/Wifi.png" alt="Wifi Connection">
+          <img src="img/Battery.png" alt="Battery">
+        </div>
+      </div>
+      <div class="toolBar">
+        <!-- Use div "logo" for any future change to use an image. -->
+        <div id="logo">
+          <h1>J. Luis Arredondo</h1>
+        </div>
+        <nav class="menu">
+          <img id="menuBar" src="img/menu.png" alt="Menu">
+        </nav>
+      </div>
+    </header>
+
+    <main>
+      <h2 id="intro">Hey There.
+      <br>I’m Alex</h2>
+      <h3 id="enphasis">I’am a Software Developer</h3>
+      <p id="landingText">
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+        <br>don’t hestiate to contact me.
+      </p>
+      <div id="socialMedia">
+        <img class="socialIco" src="img/Social_Github.png" alt="Github logo">
+        <img class="socialIco" src="img/Social_Linkedin.png" alt="Linked In logo">
+        <img class="socialIco" src="img/Social_AngelList.png" alt="Angel List logo">
+        <img class="socialIco" src="img/Social_Twitter.png" alt="Twitter logo">
+        <img class="socialIco" src="img/Social_Medium.png" alt="Medium logo">
+      </div>
```

Comment:
Kindly this is a list so it's more semantic to use `ul` tag to group them instead of `div`, and `li` per icon, and each icon has a link (`a` tag).
Like 
```
<ul>
     <li><a href=".."><img  ...></a></li>
      ......
</ul>
```